src/lib/Libwin/util.c

```diff
@@ -744,7 +704,6 @@ win_open(const char *filename, int oflag, ...)
 	va_end(vl);
 
 	if ((err) || (fd == -1)) {
-		errno = err;
```

Comment:
Why are we removing this? Caller of open() and create() in Windows may check errno for validation